demoScripts/vmssQuickstart.md

```diff
@@ -0,0 +1,254 @@
+---
+
+# Quickstart: Create a Virtual Machine Scale Set with the Azure CLI
+
+**Applies to:** :heavy_check_mark: Linux VMs :heavy_check_mark: Windows VMs :heavy_check_mark: Uniform scale sets
+
+> [!NOTE]
+> The following article is for Uniform Virtual Machine Scale Sets. We recommend using Flexible Virtual Machine Scale Sets for new workloads. Learn more about this new orchestration mode in our [Flexible Virtual Machine Scale Sets overview](flexible-virtual-machine-scale-sets.md).
+
+A Virtual Machine Scale Set allows you to deploy and manage a set of auto-scaling virtual machines. You can scale the number of VMs in the scale set manually, or define rules to autoscale based on resource usage like CPU, memory demand, or network traffic. An Azure load balancer then distributes traffic to the VM instances in the scale set. In this quickstart, you create a Virtual Machine Scale Set and deploy a sample application with the Azure CLI.
+
+[!INCLUDE [quickstarts-free-trial-note](../../includes/quickstarts-free-trial-note.md)]
+
+[!INCLUDE [azure-cli-prepare-your-environment.md](~/articles/reusable-content/azure-cli/azure-cli-prepare-your-environment.md)]
+
+- This article requires version 2.0.29 or later of the Azure CLI. If using Azure Cloud Shell, the latest version is already installed. 
+
+
+## Define Environment Variables
+
+```azurecli-interactive
+export RESOURCE_GROUP_NAME=myResourceGroup
+export RESOURCE_LOCATION=eastus
+export SCALE_SET_NAME=myScaleSet 
+export BASE_VM_IMAGE=UbuntuLTS
+export ADMIN_USERNAME=azureuser
+export LOAD_BALANCER_NAME=myScaleSetLB 
+export BACKEND_POOL_NAME=myScaleSetLBBEPool
+export LOAD_BALANCER_RULE_NAME=myLoadBalancerRuleWeb
+export FRONT_END_IP_NAME=loadBalancerFrontEnd 
+export CUSTOM_SCRIPT_NAME=customScript
+export SCALE_SET_PUBLIC_IP=myScaleSetLBPublicIP
+```
+
+## Create a scale set
+Before you can create a scale set, create a resource group with [az group create](/cli/azure/group). The following example creates a resource group named *myResourceGroup* in the *eastus* location:
+
+```azurecli-interactive
+az group create --name $RESOURCE_GROUP_NAME --location $RESOURCE_LOCATION
+```
+
+<!--expected_similarity=0.2-->
+```Output
+{
+  "id": "/subscriptions/<guid>/resourceGroups/myResourceGroup",
+  "location": "eastus",
+  "managedBy": null,
+  "name": "myResourceGroup",
+  "properties": {
+    "provisioningState": "Succeeded"
+  },
+  "tags": null,
+  "type": "Microsoft.Resources/resourceGroups"
+}
+```
+
+Now create a Virtual Machine Scale Set with [az vmss create](/cli/azure/vmss). The following example creates a scale set named *myScaleSet* that is set to automatically update as changes are applied, and generates SSH keys if they do not exist in *~/.ssh/id_rsa*. These SSH keys are used if you need to log in to the VM instances. To use an existing set of SSH keys, instead use the `--ssh-key-value` parameter and specify the location of your keys.
+
+```azurecli-interactive
```

Comment:
Needs a result test as the next step cannot be carried out until this one has completed and the VMSS is available.